axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/SaleOrderServiceSupplychainImpl.java

```diff
@@ -138,6 +136,7 @@ public void updateAmountToBeSpreadOverTheTimetable(SaleOrder saleOrder) {
   @Transactional(rollbackOn = {Exception.class})
   public boolean enableEditOrder(SaleOrder saleOrder) throws AxelorException {
     boolean checkAvailabiltyRequest = super.enableEditOrder(saleOrder);
+    AppSupplychain appSupplychain = appSupplychainService.getAppSupplychain();
 
     if (!Beans.get(AppSupplychainService.class).isApp("supplychain")) {
```

Comment:
```suggestion
    if (!appSupplychainService.isApp("supplychain")) {
```

Check the rest of the file to do similar replacements if relevant.

Reply:
Done.